faker/providers/address/vi_VN/__init__.py

```diff
@@ -0,0 +1,313 @@
+# coding=utf-8
+
+from __future__ import unicode_literals
```

Comment:
this import should not be there anymore sincePython 2.7 is not supported anymore.
It was already done in the other providers files 2 days ago in https://github.com/laurentbristiel/faker/commit/6e4a09eb381b3a6fa055c0b47f61dee1c13d9754

Reply:
@magiskboy Can you please delete this import?